### DISCUSSION TITLE
Reasoning error--KeyError: 'xxx is not in the models registry'

### DISCUSSION THREAD

**Discussion by SuperFDX** [obss/sahi#834] [Q&A] [answered]
@fcakyon Hello, when I use sahi-main/demo/reference_ for_ The following error occurred when mmdetection.ipynb reasoned the local model modified based on the mmdetection library. What should I modify to allow sahi to support my modified local model.
Error:KeyError: 'xxx is not in the models registry'
'xxx' is my own local model modified based on mmdetection library.

**Comment by SuperFDX** [comment]
@fcakyon Hello, look forward to your reply

**Comment by fcakyon** [comment, marked as answer]
It seems like a mmdet issue. You should open an issue on mmdetection repo.
